.github/workflows/testing.yml

```diff
@@ -60,16 +60,28 @@ jobs:
         run: |
           micromamba install -n a2 -c conda-forge enumlib packmol bader --yes
 
+      - name: Install ALA-Mode
+        run: |
+          micromamba activate a2
+          micromamba install -n a2 -c conda-forge "numpy<=2.2" scipy h5py compilers “libblas=*=*mkl” spglib boost eigen cmake ipython mkl-include openmpi --yes  
+          git clone https://github.com/ttadano/ALM.git # do I need to modify this?
```

Comment:
https://anaconda.org/conda-forge/alm why not installing it directly from conda? Is a different version needed?